packages/amplify-ui-components/src/components/amplify-form-section/amplify-form-section.tsx

```diff
@@ -0,0 +1,39 @@
+import { Component, Prop, h } from '@stencil/core';
+import { styleNuker } from '../../common/helpers';
+import { AMPLIFY_UI_PREFIX } from '../../common/constants';
+import { formSectionFooter, formSection } from './amplify-form-section.style';
+
+const STATIC_FORM_SECTION_CLASS_NAME = `${AMPLIFY_UI_PREFIX}--form-section`;
+const STATIC_FORM_SECTION_FOOTER_CLASS_NAME = `${AMPLIFY_UI_PREFIX}--footer-section`;
+@Component({
+  tag: 'amplify-form-section',
+  shadow: false,
+})
+export class AmplifyFormSection {
+  /** (Required) Submit function used when form is submitted */
+  @Prop() handleSubmit: (inputEvent: Event) => void;
+  /** (Optional) Used as a the default value within the default footer slot */
```

Comment:
nit
```suggestion
  /** (Optional) Used as the default value within the default footer slot */
```
Also though, I think something more straightforward like "The text on the form submit button" might be better here